tests/dates/test_DateDifferenceTransformer.py

```diff
@@ -893,6 +893,841 @@ def create_datediff_test_nulls_df2(library="pandas"):
+            ),
+            datetime.datetime(
+                2018,
+                10,
+                10,
+                10,
+                59,
+                59,
+                tzinfo=datetime.timezone.utc,
+            ),
+            datetime.datetime(
+                2018,
+                12,
+                10,
+                11,
+                59,
+                59,
+                tzinfo=datetime.timezone.utc,
+            ),
+            datetime.datetime(
+                1985,
+                7,
+                23,
+                11,
+                59,
+                59,
+                tzinfo=datetime.timezone.utc,
+            ),
+        ],
+        "b": [
+            datetime.datetime(
+                2020,
+                5,
+                1,
+                12,
+                59,
+                59,
+                tzinfo=datetime.timezone.utc,
+            ),
+            datetime.datetime(
+                2019,
+                12,
+                25,
+                11,
+                58,
+                58,
+                tzinfo=datetime.timezone.utc,
+            ),
+            datetime.datetime(
+                2018,
+                11,
+                10,
+                11,
+                59,
+                59,
+                tzinfo=datetime.timezone.utc,
+            ),
+            datetime.datetime(
+                2018,
+                11,
+                10,
+                11,
+                59,
+                59,
+                tzinfo=datetime.timezone.utc,
+            ),
+            datetime.datetime(
+                2018,
+                9,
+                10,
+                9,
+                59,
+                59,
+                tzinfo=datetime.timezone.utc,
+            ),
+            datetime.datetime(
+                2015,
+                11,
+                10,
+                11,
+                59,
+                59,
+                tzinfo=datetime.timezone.utc,
+            ),
+            datetime.datetime(
+                2015,
+                11,
+                10,
+                12,
+                59,
+                59,
+                tzinfo=datetime.timezone.utc,
+            ),
+            datetime.datetime(
+                2015,
+                7,
+                23,
+                11,
+                59,
+                59,
+                tzinfo=datetime.timezone.utc,
+            ),
+        ],
+        "week": [
+            1387.571429,
+            1031.428571,
+            0.0,
+            4.428571,
+            -4.285714,
+            -152.142857,
+            -160.857143,
+            1565.285714,
+        ],
+    }
+
+    return dataframe_init_dispatch(df_dict, library=library)
+
+
+def expected_df_10(library="pandas"):
+    """Expected output for test_expected_output_units_fortnight."""
+
+    df_dict = {
+        "a": [
+            datetime.datetime(
```

Comment:
Q: Are all these datetime examples the same, differing only by the units? In which case, consider a text fixture?